File: Curso Programação Orientada a Objetos/Java com POO/Aula11/src/teste/Aluno.java
package teste;

public class Aluno extends Pessoa {
    //Atributos
    protected int matricula;
    protected String curso;
    
    //Métodos
    public void pagarMensalidade() {
        System.out.println("Pagando mensalidade do aluno " + this.nome);
    }
    
    //Getters e Setters
    //matricula
    public int getMatricula() {
        return matricula;
    }
    public void setMatricula(int matricula) {
        this.matricula = matricula;
    }

    //curso
    public String getCurso() {
        return curso;
    }
    public void setCurso(String curso) {
        this.curso = curso;
    }

    //Método toString
    @Override
    public String toString() {
        return "Aluno{" + "nome=" + nome + ", idade=" + idade + ", sexo=" + sexo + 
                " matricula=" + matricula + ", curso=" + curso + '}';
    }
        
}

File: Curso Programação Orientada a Objetos/PHP com POO/Aula11/Aluno.php
<?php

require_once 'Pessoa.php';

class Aluno extends Pessoa {
    //Atributos
    protected $matricula;
    protected $curso;
    
    //Métodos
    public function pagarMensalidade() {
        echo "<p>Pagando mensalidade do aluno " . $this->nome . "</p>";
    }
    
    //Getters
    public function getMatricula() {
        return $this->matricula;
    }

    public function getCurso() {
        return $this->curso;
    }

    //Setters
    public function setMatricula($matricula) {
        $this->matricula = $matricula;
    }

    public function setCurso($curso) {
        $this->curso = $curso;
    }


}
